commands/src/main/java/pl/fratik/commands/system/UstawPoziomCommand.java

```diff
@@ -0,0 +1,181 @@
+            embed.setColor(UserUtil.getPrimColor(context.getSender()));
+
+            for (Map.Entry<String, Integer> entry : gc.getPermLevel().entrySet()) {
+                PermLevel pl = PermLevel.getPermLevel(entry.getValue());
+                if (pl == null) {
+                    gc.getPermLevel().remove(entry.getKey());
+                    setuj = true;
+                } else {
+                    String append = context.getTranslated("ustawpoziom.list.desc",
+                            entry.getKey(),
+                            context.getTranslated(pl.getLanguageKey()),
+                            entry.getValue()) + "\n";
+                    if (sb.toString().length() + append.length() >= 900) {
+                        list.add(sb.toString());
+                        sb = new StringBuilder();
+                    } else sb.append(append);
+                }
+            }
+            if (list.isEmpty()) embed.setDescription(sb.toString());
+            else list.forEach(s -> embed.addField(" ", s, false));
+            context.send(embed.build());
+            if (setuj) guildDao.save(gc);
+            return true;
+        }
+        if (typ.equals("remove") || typ.equals("reset") || typ.equals("clear")) {
+            String cmd;
+            try {
+                cmd = ((String) context.getArgs()[1]).toLowerCase();
+            } catch (Exception e) {
+                CommonErrors.usage(context);
+                return false;
+            }
+            if (!gc.getPermLevel().containsKey(cmd)) {
+                context.send(context.getTranslated("ustawpoziom.remove.doesnt.exist"));
+                return false;
+            }
+            gc.getPermLevel().remove(cmd);
+            guildDao.save(gc);
+            context.send(context.getTranslated("ustawpoziom.remove.succes"));
+            return true;
+        }
+        if (typ.equals("set") || typ.equals("add")) {
+            String cmd;
+            Integer lvl;
+            PermLevel plvl;
+            Command ccmd = null;
+            try {
+                cmd = ((String) context.getArgs()[1]).toLowerCase();
+                lvl = (Integer) context.getArgs()[2];
+            } catch (Exception e) {
+                CommonErrors.usage(context);
+                return false;
+            }
+
+            try {
+                plvl = PermLevel.getPermLevel(lvl);
+            } catch (Exception e) {
+                context.send(context.getTranslated("ustawpoziom.set.badlvl"));
+                return false;
+            }
+            if (plvl == null) {
+                context.send(context.getTranslated("ustawpoziom.set.badlvl"));
+                return false;
+            }
+            for (Map.Entry<String, Command> entry : managerKomend.getCommands().entrySet()) {
+                if (!entry.getKey().equals(cmd)) continue;
+                ccmd = entry.getValue();
+                break;
+            }
```

Comment:
to nie wywali błędu, get zwraca nulla jak nie znajdzie

```suggestion
            ccmd = managerKomend.getCommands().get(cmd);
```